Sources/PluginLibrary/SwiftProtobufNamer.swift

```diff
@@ -81,18 +82,84 @@ public final class SwiftProtobufNamer {
     return fullName(message: containingType) + "." + relativeName
   }
 
-  /// Calculate the relative name for the given enum value.
-  public func relativeName(enumValue: EnumValueDescriptor) -> String {
-    let baseName = enumValue.name
-    if let stripped = NamingUtils.strip(protoPrefix: enumValue.enumType.name, from: baseName) {
-      let camelCased = NamingUtils.toLowerCamelCase(stripped)
-      if isValidSwiftIdentifier(camelCased) {
-        return NamingUtils.sanitize(enumCaseName: camelCased)
+  /// Compute the short names to use for the values of this enum.
+  private func computeRelativeNames(enum e: EnumDescriptor) {
+    let stripper = NamingUtils.PrefixStripper(prefix: e.name)
+
+    /// Determine the initial canidate name for the name before
```

Comment:
Woops, I didn't even realize that was a nested function—I thought it was just a comment inside this function. Nvm.